core/chains/evm/logpoller/log_poller.go

```diff
@@ -31,7 +31,7 @@ import (
 //go:generate mockery --quiet --name LogPoller --output ./mocks/ --case=underscore --structname LogPoller --filename log_poller.go
 type LogPoller interface {
 	services.ServiceCtx
-	Replay(ctx context.Context, fromBlock int64) error
+	Replay(ctx context.Context, fromBlock int64, async bool) error
```

Comment:
Did you consider a separate method?
```suggestion
	Replay(ctx context.Context, fromBlock int64) error
	ReplayAsync(fromBlock int64) error
```
It seems like it would simplify the context situation. Could it just piggy back on the other method like this?
```go
func (lp *logPoller) ReplayAsync(fromBlock int64) error {
	return lp.Replay(lp.ctx, fromBlock)
}
```